packages/contracts-bedrock/contracts/test/ResourceMetering.t.sol

```diff
@@ -134,6 +141,23 @@ contract ResourceMetering_Test is Test {
         assertEq(postBaseFee, 2125000000);
     }
 
+    function test_meter_baseFeeMaxChangeDenominator_reverts() external {
+        ResourceMetering.ResourceConfig memory rcfg = meter.resourceConfig();
+        uint64 target = uint64(rcfg.maxResourceLimit) / uint64(rcfg.elasticityMultiplier);
+        uint64 elasticityMultiplier = uint64(rcfg.elasticityMultiplier);
+        rcfg.baseFeeMaxChangeDenominator = 1;
+        meter.setParams(rcfg);
+        meter.use(target * elasticityMultiplier);
+
+        (, uint64 prevBoughtGas, ) = meter.params();
+        assertEq(prevBoughtGas, target * elasticityMultiplier);
+
+        vm.roll(initialBlockNum + 2);
+
+        vm.expectRevert("UNDEFINED");
```

Comment:
Is `UNDEFINED` like a magic string in Foundry or Solidity?

Reply:
No this is thrown by the call to `powWad` in our `Arithmetic` library since the first param will be `0`. See: https://github.com/ethereum-optimism/2023-04-optimism-judging/blob/e4321d3febf6c6dcd6cf74fd5712c268a0c2e10d/1-processed/2-medium/089.md?plain=1#L79-L87

Reply:
Added to the natspec since this isn't obvious :)